src/services/participants.service.ts

```diff
@@ -428,6 +428,53 @@ async function getEnrollmentById(
   }
 }
 
+async function participantSearch(
+  studyIdentifier: string,
+  token: string,
+  queryValue: string,
+  participantType: ParticipantActivityType,
+  searchType: 'EXTERNAL_ID' | 'PHONE_NUMBER'
+) {
+  const endpoint = constants.endpoints.participantsSearch.replace(
+    ':id',
+    studyIdentifier
+  )
+  const queryFilter =
+    participantType === 'ACTIVE'
+      ? 'enrolled'
+      : participantType === 'WITHDRAWN'
+      ? 'withdrawn'
+      : 'all'
+  const noneOfGroups = []
+  const allOfGroups = []
+  if (participantType !== 'TEST') {
+    noneOfGroups.push('test_user')
+  } else {
+    allOfGroups.push('test_user')
+  }
+  let body
+  if (searchType === 'EXTERNAL_ID') {
```

Comment:
you can get rid of some repetition here. The only difference between external_id and phone search is externalId filter or a phone filter.  So you can create the object outside the condition and then 
have an 'if' statement to set the relevant field